mathics/doc/doc.py

```diff
@@ -850,6 +850,21 @@ def __init__(self, doc):
     def __str__(self):
         return '\n'.join(str(item) for item in self.items)
 
+    def text(self, detail_level):
+        # used for introspection
```

Comment:
We should work in this before we merge the PR

Reply:
Besides this, I think we're all done in here. II haven't looked at the tests though.